pkgs/by-name/st/stem/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, fetchurl
+, python3
+}:
+
+python3.pkgs.buildPythonPackage rec {
```

Comment:
python libraries should not be added to `by-name`

https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md#limitations

Reply:
so should i be placing it in pkgs/development/libraries or another location?

Reply:
may i ask how were you intending to test this library ?

Reply:
moved to pkgs/development/tools

Reply:
how is this library mean to be used ?

Reply:
the library contains the supporting files needed to call the supporting files, python modules and the interpreter needed to execute the binary.

Reply:
how have you been testing this ? in current state it's not being referenced by anything, so it can not be even built

Reply:
you have to be anonymised, it is working, and get reference according to configuration.

Reply:
from `nixpkgs` point of view, in current form stem can not be build

eg nixpkgs-review can't see any new packages added

Reply:
ok making sense now.